tests/test_maintel_lasertracker_setup.py

```diff
@@ -0,0 +1,69 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+import logging
+import random
+import unittest
+
+from lsst.ts import standardscripts
+from lsst.ts.idl.enums import Script
+from lsst.ts.standardscripts.maintel.laser_tracker import SetUp
+
+random.seed(47)  # for set_random_lsst_dds_partition_prefix
+
+class TestSetUp(standardscripts.BaseScriptTestCase, unittest.IsolatedAsyncioTestCase):
+    async def basic_make_script(self, index):
+        self.script = SetUp(index=index)
+
+        return self.script
+
+    async def test_components(self):
+        async with self.make_script():
+            for component in self.script.group.components_attr:
+                with self.subTest(f"Check {component}", component=component):
+                    if getattr(self.script.group.check, component):
+                        assert component in self.script.components()
+
+    async def test_run(self):
+        # Start the test itself
+        async with self.make_script():
+            # Configure the script
+            await self.configure_script()
+
```

Comment:
Not sure this unit test is doing anything. Do you need some help to write it down?

Reply:
I re-wrote it, let me know what you think!